tests/DatasetsTests/filterDataset.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { expect } from "@playwright/test";
+import { test } from "../../fixture/fixture"; 
+import { 
+    filterInputLocator, 
+    filterButtonLocator, 
+    filterOptionsLocator, 
+    csvButtonLocator, 
+    applyButtonLocator, 
+    filteredResultsLocator 
+} from '../../locators/locators';
+
+
+test.describe("Kaggle datasets page", () => {
+    test.beforeEach(async ({ page }) => {
+        await page.goto('https://www.kaggle.com/datasets');
```

Comment:
I'd use `${process.env.STAGE_URL}datasets` string here - just fully use the benefits of .env file.
But prior to that - add 

import dotenv from "dotenv";
dotenv.config();

to the imports section of the test